src/main/java/seedu/address/logic/parser/RemindCommandParser.java

```diff
@@ -0,0 +1,33 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import seedu.address.commons.util.StringUtil;
+import seedu.address.logic.commands.RemindCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.person.ReminderDatePredicate;
+
+/**
+ * Parses input argument and creates a new RemindCommand object
+ */
+public class RemindCommandParser {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the FindCommand
+     * and returns a RemindCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
+     */
+
+    public RemindCommand parse(String args) throws ParseException {
+        String trimmedArg = args.trim();
+        if (trimmedArg.isEmpty()) {
+            throw new ParseException(
+                    String.format(MESSAGE_INVALID_COMMAND_FORMAT, RemindCommand.MESSAGE_USAGE));
+        }
+        if (!StringUtil.isUnsignedInteger(trimmedArg)) {
```

Comment:
You can maybe merge this two if you are not trying to differentiate the reason why there is a parse failure (since you are throwing the same exception). In a way the error message will be less meaningful but it is what the original coder(s) did too for AddCommandParser etc. We discussed this briefly during our last meeting when @pPris brought it up. I think for now we are KIV-ing and maybe changing it if we have time.